kvrocks.conf

```diff
@@ -911,3 +930,5 @@ rocksdb.rate_limiter_auto_tuned yes
 
 ################################ NAMESPACE #####################################
 # namespace.test change.me
+backup-dir /tmp/kvrocks/backup
+pidfile /tmp/kvrocks/kvrocks.pid
```

Comment:
These two lines should not be committed.

Reply:
Yes, I will take a look at why those two lines are generated after running.

Reply:
Removed!